docs/release-notes/RELEASE-NOTES-5.11.0.md

```diff
@@ -0,0 +1,33 @@
+# About this release
+
+[This](https://github.com/aeternity/aeternity/releases/tag/v5.11.0) is a maintenance Lima release.
+
+It:
+
+* Fixes an issue where some nodes are stuck and could not sync beyond a
+  certain point.
+
+* Exports a function (`aec_chain_state:grant_fees/5`) in order to be used in
+  the MDW while doing dry runs.
+
+* Improves the transaction pool to not allow in `paying_for` transactions that
```

Comment:
Well until Iris the Tx-pool will reject them anyway :-)

Reply:
Indeed but it felt like wrong adding it to `iris` release notes anyway

Reply:
I believe you always say "not to" instead of "to not"